packages/common/src/ee/AiAgent/chartConfig/web/runQueryTool/viz/bar.ts

```diff
@@ -57,7 +57,9 @@ export const getBarChartConfig = ({
                     const defaultProperties = {
                         type: CartesianSeriesType.BAR,
                         yAxisIndex: 0,
-                        // ...(chartConfig?.stackBars && { stack: 'total' }),
+                        ...(chartConfig?.stackBars && {
+                            stack: 'stack-all-series',
+                        }),
```

Comment:
So we used to directly set `stack: boolean` in the series config, I assume now that pivot is handled by a different function, it will set it for us?

https://github.com/lightdash/lightdash/blob/facf7cfcbaf0fcf334c170e62b510fa917bbd7a3/packages/
frontend/src/ee/features/aiCopilot/utils/echarts.ts#L116

Reply:
well this is a great question. With the introduction of `stackBoolean` from 3h ago it might be very different now. Checking

Reply:
yea, I've simplified this logic a lot more now. 
Ignore my previous comment, that property isn't suitable for our use-case.

LMK what you think of the clearing up. We don't need to format pivot value, bc then that will be handled differently